hoomd/md/TwoStepRATTLENVE.h

```diff
@@ -51,25 +51,44 @@ template<class Manifold> class PYBIND11_EXPORT TwoStepRATTLENVE : public Integra
     virtual ~TwoStepRATTLENVE();
 
     //! Sets the movement limit
-    void setLimit(Scalar limit)
+    void setLimit(pybind11::object limit)
         {
-        m_limit = true;
-        m_limit_val = limit;
+        if (limit.is_none())
+            {
+            m_limit = false;
+            }
+        else
+            {
+            m_limit = true;
+            m_limit_val = pybind11::cast<Scalar>(limit);
+            }
         }
 
-    //! Removes the limit
-    void removeLimit()
+    pybind11::object getLimit()
         {
-        m_limit = false;
+        pybind11::object result;
+        if (m_limit)
+            {
+            result = pybind11::cast(m_limit_val);
+            }
+        else
+            {
+            result = pybind11::none();
+            }
+        return result;
         }
 
-    //! Sets the zero force option
-    /*! \param zero_force Set to true to specify that the integration with a zero net force on each
-       of the particles in the group
-    */
-    void setZeroForce(bool zero_force)
+    void setZeroForce(pybind11::object zero_force)
         {
-        m_zero_force = zero_force;
+        m_zero_force = pybind11::cast<bool>(zero_force);
+        }
+
+    pybind11::object getZeroForce()
```

Comment:
Does this need to return `pybind11::object`? What its actually returning is a boolean, so the code would be easier to read and understand if it returned `pybind11::bool`.

Reply:
Taking this a step further, there is no need for `pybind11::object` or `pybind11::bool` - just use `bool`. Same for the `setZeroForce` method. Let pybind11 handle the conversions.